manifests/01-asb-catalogsource-configmap.yaml

```diff
@@ -0,0 +1,367 @@
+                verbs:
+                - create
+              - apiGroups:
+                - authorization.k8s.io
+                resources:
+                - subjectaccessreviews
+                verbs:
+                - create
+              - apiGroups:
+                - rbac.authorization.k8s.io
+                resources:
+                - rolebindings
+                verbs:
+                - create
+                - delete
+              # broker must be able to set up a networkpolicy in the target
+              # namespace to set up a tunnel between the sandbox and the target
+              - apiGroups:
+                - networking.k8s.io
+                resources:
+                - networkpolicies
+                verbs:
+                - create
+                - delete
+              - apiGroups:
+                - authorization.openshift.io
+                resources:
+                - subjectrulesreview
+                verbs:
+                - create
+              - apiGroups:
+                - image.openshift.io
+                - ""
+                resources:
+                - images
+                verbs:
+                - get
+                - list
+              - apiGroups:
+                - network.openshift.io
+                - ""
+                resources:
+                - clusternetworks
+                - netnamespaces
+                verbs:
+                - get
+              - apiGroups:
+                - network.openshift.io
+                - ""
+                resources:
+                - netnamespaces
+                verbs:
+                - update
+            - serviceAccountName: automation-broker-client
+              rules:
+              - nonResourceURLs:
+                - "/osb"
+                - "/osb/*"
+                verbs:
+                - get
+                - post
+                - put
+                - patch
+                - delete
+            deployments:
+            - name: automation-broker-operator
+              spec:
+                replicas: 1
+                selector:
+                  matchLabels:
+                    name: automation-broker-operator-alm-owned
+                template:
+                  metadata:
+                    name: automation-broker-operator-alm-owned
+                    labels:
+                      name: automation-broker-operator-alm-owned
+                  spec:
+                    serviceAccountName: automation-broker-operator
+                    containers:
+                    - name: automation-broker-operator
+                      image: docker.io/automationbroker/automation-broker-operator:latest
+                      imagePullPolicy: IfNotPresent
+                      env:
+                      - name: IMAGE
+                        value: docker.io/ansibleplaybookbundle/origin-ansible-service-broker:latest
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
```

Comment:
I believe we should hardcode the `WATCH_NAMESPACE` to be the namespace where we want the broker deployed into like `openshift-ansible-service-broker`